src/components/header/SubHeader.js

```diff
@@ -218,6 +224,9 @@ class Subheader extends Component {
             } while (currentNode && currentNode.children && (currentNode.type !== 'window'));
         }
 
+        // TODO: refine gridView conditional
+        const gridView = parseInt(windowType) === 143;
```

Comment:
atm i would suggest to not enforce it at all. Just let it be enabled for any view.

Reply:
Another option would be to only enable the button if `selected.length > 0`. Or do you want the button the be there when nothing is selected (better discoverabilty of that feature)?